tests/benchmarks/enforcer_cached_b.cpp

```diff
@@ -60,53 +60,7 @@ static void BenchmarkCachedRBACModelSmall(benchmark::State& state) {
         e.Enforce(params);
 }
 
-BENCHMARK(BenchmarkCachedRBACModelSmall);
```

Comment:
I think commenting this out is better than deleting it right away. If we comment out only this specific line, the benchmark for `BenchmarkCachedRBACModelSmall` won't run. However, the user has the option to run it anyway.

Reply:
What do you think about this @noob20000405?

Reply:
@EmperorYP7 Totally agree with you, I'll change it now.